pkg/monitortests/node/watchnodes/node.go

```diff
@@ -186,12 +188,20 @@ func startNodeMonitoring(ctx context.Context, m monitorapi.RecorderWriter, clien
 			now := time.Now()
 			if !isOldNodeUnReachable && isNewNodeUnReachable && !isNodeUnscheduable {
```

Comment:
```go

// always mark unreachable
if !isOldNodeUnReachable && isNewNodeUnReachable{
	intervals = append(intervals,
		monitorapi.NewInterval(monitorapi.SourceUnreachable, monitorapi.Info).
			Locator(monitorapi.NewLocator().NodeFromName(node.Name)).
			Message(monitorapi.NewMessage().Reason(monitorapi.NodeUnreachable).
				HumanMessage("node unreachable")).
			Display().
			Build(now, now))
}

// sometimes also create the unexpectedunreachable
if !isOldNodeUnReachable && isNewNodeUnReachable && !isNodeUnscheduable {
	intervals = append(intervals,
		monitorapi.NewInterval(monitorapi.SourceUnreachable, monitorapi.Error).
			Locator(monitorapi.NewLocator().NodeFromName(node.Name)).
			Message(monitorapi.NewMessage().Reason(monitorapi.NodeUnexpectedUnreachableReason).
				HumanMessage("unexpected node unreachable")).
			Display().
			Build(now, now))
}
```

this is peer to the always create node not ready, sometimes also create the unexpectednotready.

Reply:
updated.